### PR TITLE
MUMUP-2440 : General cleanup of priority notifications on notification page

### PR DESCRIPTION
**Pull request by timlevett**
+ Don't show priority banner on notification page
+ Show ! triangle and highlight the border with color1 for priority notifications 

http://goo.gl/wp02cJ

### CODE REVIEW

File: uw-frame-components/portal/notifications/controllers.js [hunk 1]
```diff
@@ -6,31 +6,64 @@ define(['angular'], function(angular) {
...
+      $scope.priorityNotifications = [];
+      $scope.hasPriorityNotifications = false;
+      if($scope.headerCtrl) {
+        $scope.headerCtrl.hasPriorityNotifications = false;
+      }
+      $('.page-content').removeClass('has-priority-nots');
+    }
+
+    $scope.$on('$locationChangeStart', function(event) {
+      if ($location.url().indexOf('notification') === -1) {
```
Comment (apetro) [inline uw-frame-components/portal/notifications/controllers.js]: Does this mean that when viewing content with `fname` `some-notification`, the priority notification header would be suppressed? How about if we invented a `/settings/notifications`?

**Comment by apetro** [comment]
:+1:

File: uw-frame-components/css/buckyless/notifications.less [hunk 2]
```diff
@@ -13,6 +13,10 @@
     border-radius:6px;
     margin:8px 10px;
   }
+  .notifications-list .notification-item.priority {
+    border:2px solid @color1;
```
Comment (jhanstra) [inline uw-frame-components/css/buckyless/notifications.less]: a wee bit more padding on the right side of the exclamation icon would be good

Comment (timlevett) [inline uw-frame-components/css/buckyless/notifications.less]: Agreed, will adjust

Comment (timlevett) [inline uw-frame-components/css/buckyless/notifications.less]: Adjusted 5px

http://goo.gl/myYIv6

**Comment by vertein** [comment]
:+1:

**Comment by jhanstra** [comment]
:+1:

**Comment by keirserrie** [comment]
:+1:

**Pull request merged**